projects/program-website/src/app/app.component.html

```diff
@@ -1,6 +1,17 @@
 <div *ngIf="!book">
   Loading
 </div>
+<nav class="pull-right">
+  <a routerLink="/projects">  Projects  </a>
+  <a routerLink="/participants">  Participants  </a>
+  <a routerLink="/about"> About </a>
```

Comment:
Why did you add space here? `Projects` has even 2 spaces.
FYI: html ignore more than 1 space.

Why not:
```html
<a routerLink="/projects">Projects</a>
```
If you tried to separate menu options, then it's be better to use css for the purpose.

Reply:
Handled.